app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    _, file, copy = command.split(" ")
```

Comment:
what if command is not cp?

Reply:
@S-SMTN This tasks requires manual testing. So please fix comments